presentation/src/main/java/com/anytypeio/anytype/presentation/notifications/NotificationsProvider.kt

```diff
@@ -0,0 +1,77 @@
+package com.anytypeio.anytype.presentation.notifications
+
+import com.anytypeio.anytype.core_models.Notification
+import com.anytypeio.anytype.domain.account.AwaitAccountStartManager
+import com.anytypeio.anytype.domain.base.AppCoroutineDispatchers
+import com.anytypeio.anytype.domain.workspace.NotificationsChannel
+import javax.inject.Inject
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.launch
+import timber.log.Timber
+
+interface NotificationsProvider {
+    fun start()
+    fun stop()
+    fun get(): Notification?
+    fun observe(): Flow<Notification?>
+
+    class Default @Inject constructor(
+        private val dispatchers: AppCoroutineDispatchers,
+        private val scope: CoroutineScope,
```

Comment:
are you using app scope here?

Reply:
@Named(ConfigModule.DEFAULT_APP_COROUTINE_SCOPE) scope: CoroutineScope, this one